app/vmalert/config/config_test.go

```diff
@@ -15,13 +15,13 @@ import (
 )
 
 func TestMain(m *testing.M) {
-	u, _ := url.Parse("https://victoriametrics.com/path")
-	notifier.InitTemplateFunc(u)
+	tmpl, _ := notifier.LoadTemplates([]string{"testdata/*good.tmpl"})
```

Comment:
Looks like there is no templates in this path `"testdata/*good.tmpl"`, so it loads nothing